ui/src/components/GeneralStatic.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from "react";
+import { FormattedMessage } from "react-intl";
+
+/**
+ * General static component for rendering pagesS
+ *
+ * @class StaticComp
+ * @extends {Component}
+ */
+const StaticComp = ({ subtitle, children }) => (
+  <div>
+    <div id='repoheader-info'>
+      <p className='mdc-typography--subtitle1'>{subtitle}</p>
+    </div>
+    { children }
```

Comment:
Please wrap the children in a div with a unique class name (`staticpage-contents` might be a good choice). That way we can apply spacing or stuff like that we might need.

Reply:
I think we'll want to include the footer on every page, so please include it after the children (the footer component is in `components/Footer.js`).